api/app/services/pdfs/expenses_overview_service.rb

```diff
@@ -0,0 +1,220 @@
+      { content: 'Fr.', background_color: COLOR_GREY, align: :right },
+      { content: 'Fr.', background_color: COLOR_GREY, align: :right },
+      { content: 'Tage', background_color: COLOR_GREY, align: :left },
+      { content: 'Fr.', background_color: COLOR_GREY, align: :right },
+      { content: 'Fr.', background_color: COLOR_GREY, align: :right },
+      { content: 'Tage', background_color: COLOR_GREY, align: :right },
+      { content: 'Fr.', background_color: COLOR_GREY, align: :right }
+    ].freeze
+
+    def initialize(service_specifications, dates)
+      @beginning = dates.beginning
+      @ending = dates.ending
+      @service_specifications = service_specifications
+
+      update_font_families
+
+      header
+      content_table
+    end
+
+    def document
+      @document ||= Prawn::Document.new(page_size: 'A4', page_layout: :landscape)
+    end
+
+    private
+
+    def header
+      text I18n.t('pdfs.expenses_overview.swo', date: I18n.l(Time.zone.today)), align: :right, size: 8
+      text I18n.t('pdfs.expenses_overview.basedon', date: I18n.l(Time.zone.today)), align: :right, size: 8
+      text(
+        I18n.t(
+          'pdfs.expenses_overview.title',
+          beginning: I18n.l(@beginning),
+          ending: I18n.l(@ending)
+        ),
+        align: :left,
+        style: :bold,
+        size: 15
+      )
+    end
+
+    def content_table
+      font_size 9
+      table([TABLE_HEADER, TABLE_SUB_HEADER],
+            cell_style: { borders: %i[] },
+            width: bounds.width,
+            header: true,
+            column_widths: [40, 90, 80, 30, 35, 30, 35, 30, 35, 30, 35, 30, 35, 60, 30, 30, 40, 30]) do
+        row(0).font_style = :bold
+      end
+      @service_specifications.each do |_name, expense_sheets|
+        table(table_data(expense_sheets),
+              cell_style: { borders: %i[] },
+              width: bounds.width,
+              header: true,
+              column_widths: [40, 90, 80, 30, 35, 30, 35, 30, 35, 30, 35, 30, 35, 60, 30, 30, 40, 30]) do
+        end
+
+        table([[{ content: 'Gesamt: ', align: :left },
+                { content: (expense_sheets.sum(&:work_days) + expense_sheets.sum(&:workfree_days)).to_s,
+                  align: :right },
+                { content: Pdfs::ExpenseSheet::FormatHelper.to_chf(expense_sheets.sum(&:calculate_full_expenses).to_s),
+                  align: :right }]], cell_style: { borders: %i[] },
+                                     header: false,
+                                     position: :right,
+                                     column_widths: [40, 30, 45]) do
+          row(0).font_style = :bold
+        end
+      end
+    end
+
+    def pre_table(name)
+      move_down 25
+
+      text(
+        name,
+        align: :left,
+        style: :bold,
+        size: 11
+      )
+    end
+
+    def table_data(expense_sheets)
+      move_down 10
+      [].push(*table_content(expense_sheets))
```

Comment:
Wieso genau pushish du ines leers Array? Efach 
```suggestion
      table_content(expense_sheets)
```
müssti doch au gah